packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -4655,4 +4723,127 @@ describe('TransactionController', () => {
       expect(stopSpy).toHaveBeenCalledTimes(1);
     });
   });
+  describe('initTrackingMap', () => {
+    // eslint-disable-next-line jest/no-done-callback
+    it('should initialize the tracking map on construction', (done) => {
+      const hub = new EventEmitter() as TransactionControllerEventEmitter;
+      hub.on('tracking-map-init', () => {
```

Comment:
i don't have a good alternative, but I'm unsure if we should be using the messaging hub only as a way to probe the transaction controller in tests

Reply:
should assert on the networkClientIds being initialized

Reply:
Agreed. I suggested it would be ok if we add tests of the actual workings of the trackingMap in a subsequent PR

Reply:
done here https://github.com/MetaMask/core/pull/3770/commits/abdb8965acbd21d8e131cdc2cc1fb7e50db6d469